docs/cfg.md

```diff
@@ -148,7 +149,6 @@ validation dataset and to detect and prevent overfitting.
 
 | Key         | Value | Description                                                                 |
 |-------------|-------|-----------------------------------------------------------------------------|
-| val         | True  | validate/test during training                                               |
```

Comment:
@AyushExel BTW I moved this line to training section as it belongs a training arg.

Reply:
okay sounds good